docs/services/firecrest.md

```diff
@@ -1,37 +1,88 @@
 
-Please refer to the [FirecREST documentation](https://firecrest.readthedocs.io/en/latest/) for detailed documentation.
 
 ## FirecREST Deployment on Alps
 
-FirecREST is available for all three major [Alps platforms][ref-alps-platforms], with a different API endpoint for each platform.
+FirecREST is available for all three major [Alps platforms][ref-alps-platforms], with a different API endpoint and versions for each platform.
+
+<table>
+<tr><th>Platform</th><th>Version</th><th>API Endpoint</th><th>Clusters</th></tr>
+<tr><td style="vertical-align: middle;" rowspan="2">HPC Platform</td><td>v1</td><td>https://api.cscs.ch/hpc/firecrest/v1</td><td style="vertical-align: middle;" rowspan="2"><a href="../../clusters/daint">Daint</a>, <a href="../../clusters/eiger">Eiger</a></td></tr>
+<tr>                                 <td>v2</td><td>https://api.cscs.ch/hpc/firecrest/v2</td></tr>
+<tr><td>ML Platform</td><td>v1</td><td>https://api.cscs.ch/ml/firecrest/v1</td><td style="vertical-align: middle;"><a href="../../clusters/bristen">Bristen</a>, <a href="../../clusters/clariden">Clariden</a></td></tr>
+<tr><td style="vertical-align: middle;" rowspan="2">CW Platform</td><td>v1</td><td>https://api.cscs.ch/cw/firecrest/v1</td><td style="vertical-align: middle;" rowspan="2"><a href="../../clusters/santis">Santis</a></td></tr>
+<tr><td>v2</td><td>https://api.cscs.ch/cw/firecrest/v2</td></tr>
+</table>
+
 
-| Platform | API Endpoint | Clusters |
-|----------|-------------|-----------|
-| HPC Platform | https://api.cscs.ch/hpc/firecrest/v1 | [Daint][ref-cluster-daint], [Eiger][ref-cluster-eiger] |
-| ML Platform | https://api.cscs.ch/ml/firecrest/v1 | [Bristen][ref-cluster-bristen], [Clariden][ref-cluster-clariden] |
-| CW Platform | https://api.cscs.ch/cw/firecrest/v1 | [Santis][ref-cluster-santis] |
+## Accessing FirecREST
 
-## Developer Portal: Creating FirecREST clients
+### Clients and Access tokens
 
-The [Developer Portal](https://developer.cscs.ch) is an application based on [WSO2 API Manager](https://wso2.com/api-manager/) that facilitates CSCS Users to manage subscriptions to an API at CSCS (such as FirecREST).
+For authenticating requests to FirecREST, **client applications** use an **access token** instead of directly using the user's credentials.
+The access token is a signed JSON Web Token ([JWT](https://jwt.io/introduction)) which contains user information and is only valid for a short time (5 minutes).
+Behind the API, all commands launched by the client will use the account of the user that registered the client, inheriting their access rights.
 
-Start by navigating to [developer.cscs.ch](https://developer.cscs.ch), then sign in by clicking the "SIGN-IN" button on the top right hand corner of the page.
+Every client will have a client ID (Consumer Key) and a secret (Consumer Secret) that will be used to get a short-lived access token with an HTTP request.
+
+!!! example "curl call to fetch the access token"
```

Comment:
```suggestion
??? example "`curl` call to fetch the access token"
```
Collapse the example since it has a clear title? This might just be personal preference though... Otherwise make it a tip?